sklearn/linear_model/cd_fast.pyx

```diff
@@ -218,9 +257,8 @@ def enet_coordinate_descent(np.ndarray[DOUBLE, ndim=1] w,
 
                 if w[ii] != 0.0:
                     # R -=  w[ii] * X[:,ii] # Update residual
-                    daxpy(n_samples, -w[ii],
-                          <DOUBLE*>(X.data + ii * n_samples * sizeof(DOUBLE)),
-                          1, <DOUBLE*>R.data, 1)
+                    axpy(n_samples, -w[ii], &X_data[ii * n_samples], 1,
```

Comment:
I removed just this line and replaced it with

```python
R -= w[ii] * X[:, ii]
```

and it works as it is supposed to :/